### ISSUE TITLE
Make setup process more friendly for user

### ISSUE THREAD

**Issue by klesh** [merico-dev/lake#372]
## Description

The current user setup process is like:

1. Clone repo (git)
2. Configure `.env` file (devlake specific config format)
3. Launch app by docker-compose
4. Send request by  curl/postman to trigger plugins (curl/postman/api calling)
5. Observe docker log for all work to finish (docker-compose logs -f devlake)
6. Open granfana web ui by copy and paste URL we provided (tedious)

The process involve deep knowledge of our tech stack, and it's tedious.  Require certain level of problem solving skill.

I would like to proposal a more user-friendly process for no-so-technical person, and this will also make things easier for technical person to try out our product. 

## Describe the solution you'd like

1. Download or copy `docker-compose.yml`
2. Launch app by docker-compose
3. Open `config-ui` setup all plugins, trigger and observe the progress.
4. Click grafana link on `config-ui` to see the charts.

TLDR: user run only one command line `docker-compose up -d` and then finish rest on our `config-ui`

## Has the Feature been Requested Before?

No

## Additional context

This proposal is for Phase 3, feel free to comment and share your thoughts.

**Comment by hezyin** [comment]
90% of this proposal has been implemented.
